packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -605,6 +605,20 @@ const replaceNodeHtmlImages = async ({
               reporter,
               cache,
             })
+
+            const webpResult = await fluid({
+              file: fileNode,
+              args: {
+                maxWidth,
+                quality,
+                pathPrefix,
+                toFormat: `WEBP`,
+              },
+              reporter,
+              cache,
+            })
```

Comment:
This will need to be behind a plugin option with its default set to disabled since this is otherwise a breaking change. We'll also need an addition to the docs for this 👍

Reply:
Just chatted with @wardpeet and we're thinking the default for the option should be enabled. As long as we have a good place in the docs explaining how to disable this we should be good.